cmd/minishift/cmd/start.go

```diff
@@ -198,14 +199,27 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
-	if err := openshift.AddSudoersRoleForUser("developer"); err != nil {
+	if err := oc.AddSudoerRoleForUser("developer"); err != nil {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
-	if err := openshift.AddContextForProfile(constants.MachineName, ip, "developer", "myproject"); err != nil {
+
+	if err := oc.AddContextForProfile(constants.MachineName, ip, "developer", "myproject"); err != nil {
```

Comment:
we wil see. We need to go away from constants. They are not going to fly long term. Maybe temporary we can do it. We'll see